modules/nf-core/openms/idscoreswitcher/main.nf

```diff
@@ -36,7 +37,7 @@ process OPENMS_IDSCORESWITCHER {
 
     stub:
     def args = task.ext.args ?: ''
-    def prefix = task.ext.prefix ?: "${meta.id}_score_switched"
+    def prefix = task.ext.prefix ?: "${meta.id}"
```

Comment:
Please also add the check to the stub :) The stub is meant to mimic the real process as close as possible without running the tool